From d753a5e508b594a046e78d7e6df26ec4bea7aff0 Mon Sep 17 00:00:00 2001
From: Yu Yue <yuyue@mprc.pku.edu.cn>
Date: Tue, 1 Nov 2011 15:33:18 +0800
Subject: [PATCH 059/641] UniCore64: close config VGA_CONSOLE

Fixed error:
hid-input.c:(.text+0x838): undefined reference to `screen_info'

Signed-off-by: Yu Yue <yuyue@mprc.pku.edu.cn>
---
 arch/unicore64/configs/unicore64_defconfig |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/unicore64/configs/unicore64_defconfig b/arch/unicore64/configs/unicore64_defconfig
index 15cf93f..da6a04d 100644
--- a/arch/unicore64/configs/unicore64_defconfig
+++ b/arch/unicore64/configs/unicore64_defconfig
@@ -6,3 +6,6 @@ CONFIG_LOCALVERSION="-uc64"
 CONFIG_ARCH_FPGA=y
 CONFIG_CPU_ICACHE_DISABLE=y
 CONFIG_CPU_DCACHE_DISABLE=y
+
+#	Console display driver support
+CONFIG_VGA_CONSOLE=n
-- 
1.7.9.5

